deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml

```diff
@@ -32,3 +32,22 @@ spec:
     rootDevices: ['0000:02:00.0', '0000:02:00.1']
   deviceType: netdevice
   isRdma: true
+
+---
+# Mlnx IB Nics doesn't support changing mtu
```

Comment:
it supported but it is diffrent from ethernet and requires configuration in opensm. Currently we can say it out of scope